### DISCUSSION TITLE
第3节课作业(11班)

### DISCUSSION THREAD

**Discussion by keesh0410** [InternLM/Tutorial#175] [作业]
🐱：提交完要记得群里的作业表哟🌹

**Comment by zhanghui-china** [comment]
基础作业：
[https://zhuanlan.zhihu.com/p/676136989](url)

**Comment by seifer08ms** [comment]
https://zhuanlan.zhihu.com/p/676719586

**Comment by qianlian-mozi** [comment]
https://github.com/qianlian-mozi/internlm_practising_20240106/tree/main/Lesson_3_honework

**Comment by Everfighting** [comment]
https://github.com/Everfighting/InternLM_learning/blob/main/chapter3/chapter_3.md

**Comment by BarryYin** [comment]
https://github.com/BarryYin/internLM/blob/main/%E7%AC%AC%E4%B8%89%E8%8A%82%E4%BD%9C%E4%B8%9A.md

**Comment by dukai289** [comment]
https://github.com/dukai289/Learning/blob/master/Learning_InternLM/Homework.md

**Comment by Shengshenlan** [comment]
https://github.com/Shengshenlan/tutorial/blob/master/homework/3/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by horacehht** [comment]
https://blog.csdn.net/m0_50123831/article/details/135498493

**Comment by wwewwt** [comment]
https://github.com/wwewwt/-InternLM/wiki/3%E3%80%81%E5%9F%BA%E4%BA%8E-InternLM-%E5%92%8C-LangChain-%E6%90%AD%E5%BB%BA%E7%9F%A5%E8%AF%86%E5%BA%93

**Comment by LichKing-W** [comment]
https://zhuanlan.zhihu.com/p/676929861

**Comment by codefd** [comment]
https://blog.csdn.net/googteo/article/details/135511255?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135511255%22%2C%22source%22%3A%22googteo%22%7D

**Comment by chenxiangzhen** [comment]
https://github.com/chenxiangzhen/InternLM-Tutorial/blob/main/%E7%AC%AC3%E8%8A%82.md

**Comment by GMYelims** [comment]
https://github.com/InternLM/tutorial/discussions/175

**Comment by wuyingxue1230** [comment]
https://zhuanlan.zhihu.com/p/677111445

**Comment by JustLike1** [comment]
https://blog.csdn.net/qq_45607888/article/details/135500463

**Comment by morning-start** [comment]
https://flowus.cn/morningstar/share/014e9726-cc06-41ba-b3dc-6a2e08cbf8bf
【FlowUs 息流】知识库作业

**Comment by Tangent-90C** [comment]
https://github.com/Tangent-90C/public-note/tree/main/%E7%AC%AC%E4%B8%89%E8%8A%82%E8%AF%BE%E7%9A%84%E4%BD%9C%E4%B8%9A

**Comment by Fulin-Guo** [comment]
https://github.com/wdnmdm/Team_learning/blob/main/InternLM/Homework3.md

**Comment by lvoooo** [comment]
https://github.com/lvoooo/internLM-learning/blob/main/3.md

**Comment by Lyusijie** [comment]
https://blog.csdn.net/weixin_44986362/article/details/135493471?spm=1001.2014.3001.5501

**Comment by hellokid123** [comment]
https://github.com/hellokid123/internllm-notes/blob/main/task/%E7%AC%AC%E4%B8%89%E8%AF%BE%E4%BD%9C%E4%B8%9A.md

**Comment by godlikeS97** [comment]
https://blog.csdn.net/godlikesun97/article/details/135535704?spm=1001.2014.3001.5501

**Comment by Kongsea** [comment]
https://blog.csdn.net/deeepsea/article/details/135453180

**Comment by Vacili-f** [comment]
https://zhuanlan.zhihu.com/p/677701571

**Comment by keesh0410** [comment]
https://zhuanlan.zhihu.com/p/679128339
